x-pack/test/lists_api_integration/security_and_spaces/tests/summary_exception_lists.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { EXCEPTION_LIST_URL, EXCEPTION_LIST_ITEM_URL } from '@kbn/securitysolution-list-constants';
+import { LIST_ID } from '../../../../plugins/lists/common/constants.mock';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { getCreateExceptionListMinimalSchemaMock } from '../../../../plugins/lists/common/schemas/request/create_exception_list_schema.mock';
+import { getCreateExceptionListItemMinimalSchemaMock } from '../../../../plugins/lists/common/schemas/request/create_exception_list_item_schema.mock';
+import { createListsIndex, deleteListsIndex, deleteAllExceptions } from '../../utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  describe('summary_exception_lists', () => {
+    describe('summary exception lists', () => {
+      afterEach(async () => {
+        await deleteListsIndex(supertest);
+        await deleteAllExceptions(es);
+      });
+      beforeEach(async () => {
+        await createListsIndex(supertest);
+      });
+
+      it('should give a validation error if the list_id and the id are not supplied', async () => {
+        const { body } = await supertest
+          .get(`${EXCEPTION_LIST_URL}/summary`)
+          .set('kbn-xsrf', 'true')
+          .send(getCreateExceptionListMinimalSchemaMock())
+          .expect(400);
+
+        expect(body).to.eql({
+          message: 'id or list_id required',
+          status_code: 400,
+        });
+      });
+
+      it('should return init summary when there are no items created', async () => {
+        const { body } = await supertest
```

Comment:
Although body is an `any` here, if you have a response schema, you can add it as a cast here and then later your typescript system will blow up if someone breaks your types.

Just a FYI, if you don't have a response schema/response types you cannot do this.